x/tally/keeper/endblock.go

```diff
@@ -80,6 +83,7 @@ func (k Keeper) ProcessTallies(ctx sdk.Context, coreContract sdk.AccAddress) err
 	processedReqs := make(map[string][]types.Distribution)
 	tallyResults := make([]TallyResult, len(tallyList))
 	dataResults := make([]batchingtypes.DataResult, len(tallyList))
+	tera_gas_burned := float32(0)
```

Comment:
Some variable naming convention issues in this file

Reply:
What is the standard then? I'm not familiar with the golang standard

Reply:
Ah The golangci comments show up in the files section of the PR